templates/deploy_fortigate_autoscale.hybrid_licensing.json

```diff
@@ -171,28 +171,28 @@
             "minValue": 0,
             "type": "Int",
             "metadata": {
-                "description": "The number of FortiGate instances the PAYG VMSS should have at any time. For High Availability in a PAYG-only use case, ensure at least 2 FortiGates are in the group. For specific use cases, set to 0 for BYOL-only, >= 2 for PAYG-only, and >= 0 for hybrid licensing."
+                "description": "Number of FortiGate instances the PAYG VMSS should have at any time. For High Availability in a PAYG-only use case, ensure at least 2 FortiGates are in the group. For specific use cases, set to 0 for BYOL-only, >= 2 for PAYG-only, and >= 0 for hybrid licensing."
             }
         },
         "PackageResURL": {
             "type": "String",
             "defaultValue": "",
             "metadata": {
-                "description": "The public URL of the function source file named fortigate-autoscale-azure-funcapp.zip, and can be found inside fortigate-autoscale-azure-template-deployment.zip. The public URL of the deployment package zip file that contains the resource used to deploy the Function App. This URL must be accessible by Azure. The default URL points to the GitHub release corresponding to this ARM template contentVersion."
+                "description": "Public URL of the deployment package fortigate-autoscale-azure-funcapp.zip. This file is within the deployment package fortigate-autoscale-azure-template-deployment.zip and contains the resource used to deploy the Function App. This URL must be accessible by Azure."
```

Comment:
@klc178 I think the existing one is better.

Reply:
specifically, the last sentence is important:
> The default URL points to the GitHub release corresponding to this ARM template contentVersion.

Reply:
```suggestion
                "description": "Public URL of the function source file fortigate-autoscale-azure-funcapp.zip. This file is within the deployment package fortigate-autoscale-azure-template-deployment.zip and contains the resource used to deploy the Function App. This URL must be accessible by Azure. The default URL points to the GitHub release corresponding to this ARM template contentVersion."
```

Reply:
This may not be the final wording for this parameter.
What I do not like about the original wording is this part:
1) The public URL of the function source file named fortigate-autoscale-azure-funcapp.zip
2) The public URL of the deployment package zip file that contains the resource used to deploy the Function App.

Reasoning. The file in (1) is inside the file in (2). File (2) has a URL, but in order to get file (1), you have to get file (2).

Reply:
> This may not be the final wording for this parameter. What I do not like about the original wording is this part:
> 
> 1. The public URL of the function source file named fortigate-autoscale-azure-funcapp.zip
> 2. The public URL of the deployment package zip file that contains the resource used to deploy the Function App.
> 
> Reasoning. The file in (1) is inside the file in (2). File (2) has a URL, but in order to get file (1), you have to get file (2).

Agree. Please update your suggestion. I will accept it.

Reply:
```suggestion
                "description": "Public URL of the Autoscale Function App source file file fortigate-autoscale-azure-funcapp.zip. This URL must be accessible by Azure. The default value points to the source file available in the release assets of the GitHub repo fortinet/fortigate-autoscale-azure."
```